packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -33,7 +33,7 @@ const systemProperties = defineProperties({
 const colorProperties = defineProperties({
   conditions: {
     lightMode: {},
-    darkMode: { selector: `.${darkThemeClass} &` },
+    darkMode: { selector: `${darkThemeClass} &` },
```

Comment:
We should leave the `.` in front of the classname. The variable only includes the class name

Reply:
OK restored this too. It's the same variable, no? 🤷  Oh, well.

Reply:
Yes, it is the same variable haha

Reply:
So this PR no longer has changes related to this, and the migrated tests are passing, so let's leave it at this for now.